### PR TITLE
Initial implement to support converting feature

### PR DESCRIPTION
**Pull request by gouniLee**
* Currently it only works `convert` feature. It doesn't support xlsxs localization file itself.
* Tested if it's work to converting files from xlsx to xliff and xlsx to xliff

### CODE REVIEW

File: XlsxFile.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+* context of the resource should match the context of
+* the file.
+*
+* @param {Resource} res a resource to add to this file
+*/
+XlsxFile.prototype.addResource = function(res) {
+    logger.trace("XlsxFile.addResource: " + JSON.stringify(res) + " to " + this.project.getProjectId() + ", " + this.locale + ", " + JSON.stringify(this.context));
+    var resLocale = res.getTargetLocale() || res.getSourceLocale();
+
+    if (res && res.getProject() === this.project.getProjectId() && resLocale === this.locale.getSpec()) {
```
Comment (ehoogerbeets) [inline XlsxFile.js:113]: This plugin only supports 1 locale at a time? This line enforces that...

File: test/assertExtras.js [hunk 2]
```diff
@@ -0,0 +1,297 @@
+/*
+ * assertExtras.js - extra assertion types to use with nodeunit
```
Comment (ehoogerbeets) [inline test/assertExtras.js]: Use the assertextras npm package for this instead and remove this one

File: test/testSuite.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var nodeunit = require("nodeunit");
+require("./assertExtras.js");
```
Comment (ehoogerbeets) [inline test/testSuite.js]:
```javascript
var nodeunit = require("nodeunit");
var assert = require("nodeunit/lib/assert");
require("assertextras")(assert);
```

Comment (ehoogerbeets) [reply test/testSuite.js]: also:
```
npm install assertextras --save-dev
```

File: XlsxFile.js (hunk 1 above)
Comment (gouniLee) [reply XlsxFile.js:113]: yes. Including this, all other webOS plugins also have the same implementation so far

File: test/testSuite.js (hunk 3 above)
Comment (gouniLee) [reply test/testSuite.js]: update it

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**